index.html

```diff
@@ -94,90 +94,13 @@ <h1 class="greeting-text">Hey there. I’m <br> Abdul Ali.</h1>
 <!-- ----------------------------------------Recent work Setion------------------------------------------- -->
 
             <section class="recent-work grid" id="recent-work">
+                
                 <div class="recent-wrok-header">
```

Comment:
-  [ ] Please the popup should be responsive fix the length of the pop and make sure they all look the same as in the design in all sizes
![image](https://user-images.githubusercontent.com/74744787/217950887-1baaf4b5-55ca-4726-9e8a-244c1f15cc37.png)